Cargo.toml

```diff
@@ -32,4 +32,5 @@ features = ["extension-module","abi3-py36"]
 
 [features]
 default = ["jemalloc"]
-jemalloc = []
\ No newline at end of file
+jemalloc = []
+no-progress-bars = []
```

Comment:
It's better to have a default enabled `progress-bars` feature I think? Then potentially in the future we can make `indicatif` optional

Reply:
This may be true.. but it is a bit of a pain to disable default features when all the packages are in the same workspace.

e.g. `cargo test --workspace` will use the default features for synapse_compress_state, even though the auto_compressor and compressor_integration_tests packages specify "default-features = false"

see https://github.com/rust-lang/cargo/issues/8366

Reply:
Ah, fair.